docs/source/get_involved/code_of_conduct.rst

```diff
@@ -168,13 +208,13 @@ GitHub.
 3.1 Contact points
 ~~~~~~~~~~~~~~~~~~
 
-Please contact the current chair of the code of conduct committee directly by email, see 4.1.
+Please contact the current chair of the Code of Conduct Committee directly by email, see 4.1.
 
 3.2 Alternate contact points
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In case of conflict of interest of sktime CoC committee members, you can report
-through the [NumFOCUS CoC process](https://numfocus.org/code-of-conduct),
+In case of conflict of interest of sktime CoC Committee members, you can report
+through the `NumFOCUS CoC process <https://numfocus.org/code-of-conduct>`__,
```

Comment:
is that actually approved by NumFOCUS? I think the did not even answer our last mails?

Reply:
in the last meeting we had with them at the CC, they said that we should not default to them, but they would act as a "last resort" if we give them jurisdiction.

My memory is a bit hazy (this PR is from July) and can't find the minutes at the moment, do you remember the date or a keyword for searching?